dashboard/app/controllers/projects_controller.rb

```diff
@@ -417,7 +417,7 @@ def remix
   end
 
   private def uses_animation_bucket?(project_type)
-    %w(gamelab spritelab).include? project_type
+    %w(gamelab poetry science spritelab story).include? project_type
```

Comment:
rather than hard-coding the subtypes here, can we pull from `standalone_app_names` in [gamelab_jr.rb](https://github.com/code-dot-org/code-dot-org/blob/9b2b9d2e60df393038c8bfe122bf5114d7478833/dashboard/app/models/levels/gamelab_jr.rb#L45) and [poetry.rb](https://github.com/code-dot-org/code-dot-org/blob/9b2b9d2e60df393038c8bfe122bf5114d7478833/dashboard/app/models/levels/poetry.rb#L68)? We will probably still need to hard-code `gamelab` here. But if we can pull from the source of truth we won't hit this again with a future subtype.

Reply:
Discussed in Slack. Part of convo in [this thread](https://codedotorg.slack.com/archives/C03DBDN67B7/p1681922735599849?thread_ts=1676479526.341989&cid=C03DBDN67B7). I will incorporate her feedback.

Reply:
I actually omitted 2 projects in my initial commit - `poetry_hoc` and `time_capsule` so this update ensures subtype list is complete as well. Thank you Molly!